### ISSUE TITLE
[Ontology] Values for regulatedEntity:regulatedEntityCategory include descriptions as part of the code

### ISSUE THREAD

**Issue by Waitak** [IATA-Cargo/ONE-Record#178]
The allowed values for regulatedEntity:regulatedEntityCategory are listed in the ontology as:

- AO - Aircraft Operator
- KC - Known Consignor
- RA - Regulated Agent
- RC - Regulated Carrier

including the descriptions as part of the code.

```
regulatedEntity:regulatedEntityCategory rdf:type owl:DatatypeProperty ;
                                        rdfs:domain :RegulatedEntity ;
                                        rdfs:range [ rdf:type rdfs:Datatype ;
                                                     owl:oneOf [ rdf:type rdf:List ;
                                                                 rdf:first "AO - Aircraft Operator" ;
                                                                 rdf:rest [ rdf:type rdf:List ;
                                                                            rdf:first "KC - Known Consignor" ;
                                                                            rdf:rest [ rdf:type rdf:List ;
                                                                                       rdf:first "RA - Regulated Agent" ;
                                                                                       rdf:rest [ rdf:type rdf:List ;
                                                                                                  rdf:first "RC - Regulated Carrier" ;
                                                                                                  rdf:rest rdf:nil
                                                                                                ]
                                                                                     ]
                                                                          ]
                                                               ]
                                                   ] ;
                                        rdfs:comment "Party type - e.g. RA - Regulated Agent, KC - Known Consignor, AO - Aircraft Operator, RC - Regulated Carrier"@en ;
                                        rdfs:label "regulatedEntity:regulatedEntityCategory"@en .
```

These should rather be just the codes (AO, KC, RA and RC):

```
regulatedEntity:regulatedEntityCategory rdf:type owl:DatatypeProperty ;
                                        rdfs:domain :RegulatedEntity ;
                                        rdfs:range [ rdf:type rdfs:Datatype ;
                                                     owl:oneOf [ rdf:type rdf:List ;
                                                                 rdf:first "AO" ;
                                                                 rdf:rest [ rdf:type rdf:List ;
                                                                            rdf:first "KC" ;
                                                                            rdf:rest [ rdf:type rdf:List ;
                                                                                       rdf:first "RA" ;
                                                                                       rdf:rest [ rdf:type rdf:List ;
                                                                                                  rdf:first "RC" ;
                                                                                                  rdf:rest rdf:nil
                                                                                                ]
                                                                                     ]
                                                                          ]
                                                               ]
                                                   ] ;
                                        rdfs:comment "Party type - e.g. RA - Regulated Agent, KC - Known Consignor, AO - Aircraft Operator, RC - Regulated Carrier"@en ;
                                        rdfs:label "regulatedEntity:regulatedEntityCategory"@en .
```

As it stands, "AO" (for example) is not a valid value for this field.

**Comment by lambertciata** [comment]
Hi it does make sense, will modify for next release

**Comment by lambertciata** [comment]
Fixed in 3.0.0 using a CodeListElement
